suite-common/wallet-core/src/accounts/accountsReducer.ts

```diff
@@ -425,9 +429,9 @@ export const selectIsDeviceNotEmpty = (
 ) => {
     const deviceState = selectDeviceState(state);
     const nonEmptyAccounts = selectNonEmptyDeviceAccounts(state);
-    const isDeviceDiscoveryActive = selectIsDeviceDiscoveryActive(state);
+    const hasDiscovery = selectDeviceDiscovery(state);
```

Comment:
the hasDiscovery variable name suggests that the value is boolean but it is actually a discovery object. Fix that please. Either by renaming the variable or creating new selector that returns boolean.

Reply:
cc10595a14

Reply:
Good idea, I will change it here and once merged expand this to all relevant places in #14267 👍

Reply:
The new `selectHasDeviceDiscovery` returns false or discovery object? I would expect boolean.

Reply:
By bad:
55a8159217